contracts/ERC721RExample.sol

```diff
@@ -82,6 +82,7 @@ contract ERC721RExample is ERC721A, Ownable {
     }
 
     function refund(uint256[] calldata tokenIds) external {
+        require(msg.sender != refundAddress, "Can't refund to refundAddress");
```

Comment:
we had another fix for this already so i think this can be removed

Reply:
Okay, I will change this with current bug fixed version